Makefile

```diff
@@ -55,6 +64,18 @@ sgx-base: sgx/Dockerfile base
 tz-base: tz/Dockerfile base
 	DOCKER_BUILDKIT=1 docker build --build-arg USER=root --build-arg UID=0 --build-arg TEE=tz -t veracruz/tz-base -f $< .
 
+.PHONY:
+nitro-base: nitro/Dockerfile base
+ifneq (,$(wildcard aws-nitro-enclaves-cli))
+	cd "aws-nitro-enclaves-cli"
+	git pull origin main
+else
+	git clone https://github.com/aws/aws-nitro-enclaves-cli.git
+endif
```

Comment:
Pinning a revision or version branch could make this part more reproducible. Not necessary of course but we have talked about striving for reproducibility.

(I'm talking about reproducibility in the informal sense, not the bit-for-bit sort we've also discussed)

Reply:
That's a good idea. Will do.

Reply:
I've done this now. Take a look